<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>737</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="Ra::Gui::Viewer" name="m_viewer" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="_statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenMesh"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFILE"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTreeView" name="m_entitiesTreeView">
              <property name="editTriggers">
               <set>QAbstractItemView::SelectedClicked</set>
              </property>
              <property name="textElideMode">
               <enum>Qt::ElideRight</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="m_removeEntityButton">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Drawables</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QListView" name="m_drawablesListView"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="m_refreshDrawableButton">
                  <property name="text">
                   <string>Refresh</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="m_toggleDrawableButton">
                  <property name="text">
                   <string>Toggle</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="m_removeDrawableButton">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Component</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Transform</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <layout class="QGridLayout" name="gridLayout_4">
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Orientation</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Position</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <layout class="QFormLayout" name="formLayout">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>X</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>Y</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_9">
                       <property name="text">
                        <string>Z</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="lineEdit_10"/>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLineEdit" name="lineEdit_11"/>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLineEdit" name="lineEdit_12"/>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="1">
                    <layout class="QFormLayout" name="formLayout_2">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_10">
                       <property name="text">
                        <string>X</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="lineEdit_13"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_11">
                       <property name="text">
                        <string>Y</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_12">
                       <property name="text">
                        <string>Z</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLineEdit" name="lineEdit_14"/>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLineEdit" name="lineEdit_15"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Material</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <layout class="QFormLayout" name="formLayout_3">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_16">
                     <property name="text">
                      <string>Rendering mode</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="m_shadersComboBox">
                     <item>
                      <property name="text">
                       <string>Default Plain Mode</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Contour Mode</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Wireframe Mode</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Diffuse</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Color</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Specular</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="lineEdit_2">
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                     <property name="cursorMoveStyle">
                      <enum>Qt::VisualMoveStyle</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QLineEdit" name="lineEdit_5">
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="3">
                    <widget class="QLineEdit" name="lineEdit_6">
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="lineEdit_7">
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="2">
                    <widget class="QLineEdit" name="lineEdit_8">
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="3">
                    <widget class="QLineEdit" name="lineEdit_9">
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>Red</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLabel" name="label_14">
                     <property name="text">
                      <string>Green</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QLabel" name="label_15">
                     <property name="text">
                      <string>Blue</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpenMesh">
   <property name="text">
    <string>Open mesh...</string>
   </property>
   <property name="toolTip">
    <string>Open a mesh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Ra::Gui::Viewer</class>
   <extends>QWidget</extends>
   <header>MainApplication/Viewer/Viewer.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
